client/my-sites/customer-home/cards/features/quick-start/index.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { Button, Card } from '@automattic/components';
+import { connect } from 'react-redux';
+import 'moment-timezone';
+
+/**
+ * Internal dependencies
+ */
+import HappinessEngineersTray from 'components/happiness-engineers-tray';
+import CardHeading from 'components/card-heading';
+import {
+	withAnalytics,
+	composeAnalytics,
+	recordTracksEvent,
+	bumpStat,
+} from 'state/analytics/actions';
+import { navigate } from 'state/ui/actions';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import QueryConciergeInitial from 'components/data/query-concierge-initial';
+import getConciergeNextAppointment from 'state/selectors/get-concierge-next-appointment';
+import { withLocalizedMoment } from 'components/localized-moment';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const QuickStart = ( {
+	moment,
+	nextSession,
+	reschedule,
+	siteId,
+	siteSlug,
+	translate,
+	viewDetails,
+} ) => {
+	return (
+		<>
+			{ siteId && <QueryConciergeInitial siteId={ siteId } /> }
+			<Card className="quick-start next-session">
+				<HappinessEngineersTray />
+				<CardHeading>{ translate( 'Your scheduled Quick Start support session:' ) }</CardHeading>
+				<table>
+					<thead>
+						<tr>
+							<th>{ translate( 'Date' ) }</th>
+							<th>{ translate( 'Time' ) }</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr>
+							<td>
+								{ nextSession
+									? moment( nextSession.beginTimestamp ).format( 'LL' )
+									: translate( 'Loading…' ) }
+							</td>
+							<td>
+								{ nextSession
+									? moment.tz( nextSession.beginTimestamp, moment.tz.guess() ).format( 'LT z' )
+									: translate( 'Loading…' ) }
+							</td>
+						</tr>
+					</tbody>
+				</table>
+				<div className="quick-start__buttons">
+					<Button disabled={ ! nextSession } onClick={ () => viewDetails( siteSlug ) }>
+						{ translate( 'View details' ) }
+					</Button>
+					<Button
+						className={ 'quick-start__reschedule' }
+						onClick={ () => reschedule( siteSlug, nextSession.id ) }
+						borderless
+						disabled={ ! nextSession }
+					>
+						{ translate( 'Reschedule' ) }
+					</Button>
+				</div>
+			</Card>
+		</>
+	);
+};
+
+export default connect(
+	( state ) => ( {
+		siteId: getSelectedSiteId( state ),
+		siteSlug: getSelectedSiteSlug( state ),
+		nextSession: getConciergeNextAppointment( state ),
+	} ),
+	( dispatch ) => ( {
+		viewDetails: ( siteSlug ) =>
+			dispatch(
+				withAnalytics(
+					composeAnalytics(
+						recordTracksEvent( 'calypso_customer_home_quick_start_view_details_click', {
+							siteSlug: siteSlug,
```

Comment:
This needs to be `site_slug` or tracks will reject the event

Reply:
Ooh, well spotted. I've updated to use the site ID, which is more consistent with other cards anyways. 9d7b5d350424aeeb474e3dcb128e900a0e4e0b66